03_SageMakerPipelinesAndDeploy/docker/Dockerfile-processing

```diff
@@ -0,0 +1,10 @@
+FROM public.ecr.aws/docker/library/r-base:4.1.2
```

Comment:
We should pin our other dockerfiles to a tag instead of latest. Will make a note